eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -142,7 +148,10 @@ public function load(array $configs, ContainerBuilder $container)
      */
     public function getConfiguration(array $config, ContainerBuilder $container)
     {
-        return new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration = new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration->setSiteAccessConfigurationFilters($this->siteaccessConfigurationFilters);
```

Comment:
That's actually what I started with. But since they actually have a default value, I chose to do it that way instead.

Are there advantages to a ctor argument ?